Makefile

```diff
@@ -12,13 +12,18 @@ deps:
 
 .PHONY: install
 install: deps
-	pip install -e .[cpu,tests,docs]
-	pip install pycocotools==2.0.0
+	pip3 install -U pip
+	pip3 install -e .[cpu]
+	pip3 install pycocotools==2.0.0
+
+.PHONY: install-dev
+install-dev: deps install
+	pip3 install -e .[test,docs]
 
 .PHONY: install-gpu
 install-gpu: install
-	pip install -e .[gpu]
-	pip install -e .[dist]
+	pip3 install -e .[gpu]
+	pip3 install -e .[dist]
 
 .PHONY: lint
 lint:
```

Comment:
Maybe we can remove the target `lint` now.

Reply:
it would be better to remain target `lint` and remove `test-blueoil-pep8`

Reply:
Fixed target names from `test-blueoil-pep8` to `test-lint` and from `test-blueoil-mypy` to `test-mypy`